roles/openshift_node/tasks/config.yml

```
@@ -70,6 +70,52 @@
     state: directory
   register: temp_dir
 
+- name: Create local temp directory
+  tempfile:
+    state: directory
+  register: local_temp_dir
+  delegate_to: localhost
+  run_once: true
+
+- name: Copy 00-runc-crc.yaml ContainerRuntimeConfig
+  copy:
+    content: |
+      apiVersion: machineconfiguration.openshift.io/v1
+      kind: ContainerRuntimeConfig
+      metadata:
+        name: openshift-ansible-use-runc
+      spec:
+        machineConfigPoolSelector:
+          matchLabels:
+            pools.operator.machineconfiguration.openshift.io/{{ openshift_node_machineconfigpool }}: ''
+        containerRuntimeConfig:
+          defaultRuntime: "runc"
+    dest: "{{ local_temp_dir.path }}/openshift-ansible-runc-crc.yaml"
+  delegate_to: localhost
+  run_once: true
+
+- name: debug cat the file
+  command: cat {{ local_temp_dir.path }}/openshift-ansible-runc-crc.yaml
+  delegate_to: localhost
+  run_once: true
```

Do you want to keep this? I lean toward dropping it, but fine to keep it.

If we keep it, let's just update the name


```suggestion
- name: Print the ContainerRuntimeConfig MachineConfig
  command: cat {{ local_temp_dir.path }}/openshift-ansible-runc-crc.yaml
  delegate_to: localhost
  run_once: true
```

---

i'll drop it.